packages/react-components/react-rating-preview/src/components/Rating/useRating.tsx

```diff
@@ -23,15 +23,14 @@ import { StarFilled, StarRegular } from '@fluentui/react-icons';
 export const useRating_unstable = (props: RatingProps, ref: React.Ref<HTMLDivElement>): RatingState => {
   const generatedName = useId('rating-');
   const {
-    appearance = props.readOnly ? 'filled' : 'outline',
-    compact = false,
+    appearance = props.mode === 'readonly' ? 'filled' : 'outline',
```

Comment:
Should you be checking for `readonly-compact` as well here?

Reply:
the api surface for this prop has changed